src/navigation-plugin.tsx

```diff
@@ -49,6 +49,7 @@ export class NavigationPlugin extends KalturaPlayer.core.BasePlugin {
   constructor(name: string, player: KalturaPlayerTypes.Player, config: NavigationConfig) {
     super(name, player, config);
     this._player = player;
+    this._activeCuePointsMap = this._makeDefaultActiveCuePointsMap();
```

Comment:
i would change it to _getDefaultActiveCuePointsMap if it returns a value

Reply:
make sense, addressed